requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-docx>=1.1.2
+python-dotenv>=1.0.1
+openai>=1.37.0
```

Comment:
don't use `>=` when pinning versions it makes it non reproducible and likely to not work at some point in the future
```suggestion
python-docx==1.1.2
python-dotenv==1.0.1
openai==1.37.0
```